packages/dds/tree/src/test/shared-tree/opSize.bench.ts

```diff
@@ -394,25 +393,17 @@ describe("Op Size", () => {
 	let currentBenchmarkName = "";
 	const currentTestOps: ISequencedDocumentMessage[] = [];
 
-	function registerOpListener(
-		tree: ISharedTree,
-		resultArray: ISequencedDocumentMessage[],
-	): void {
+	function registerOpListener(tree: ISharedTree, resultArray: ISequencedDocumentMessage[]): void {
 		// TODO: better way to hook this up. Needs to detect local ops exactly once.
-		/* eslint-disable @typescript-eslint/no-explicit-any */
 		const oldSubmitLocalMessage = (tree as any).submitLocalMessage.bind(tree);
-		function submitLocalMessage(
-			content: ISequencedDocumentMessage,
-			localOpMetadata: unknown = undefined,
-		): void {
+		function submitLocalMessage(content: any, localOpMetadata: unknown = undefined): void {
 			resultArray.push(content);
 			oldSubmitLocalMessage(content, localOpMetadata);
 		}
 		(tree as any).submitLocalMessage = submitLocalMessage;
-		/* eslint-enable @typescript-eslint/no-explicit-any */
 	}
 
-	const getOperationsStats = (operations: ISequencedDocumentMessage[]) => {
+	const getOperationsStats = (operations: ISequencedDocumentMessage[]): Record<string, any> => {
```

Comment:
This type should probably be `Record<string, unknown>` if possible.

Reply:
Thanks. Changed to type `number`.